api/src/swagger/config.yaml

```diff
@@ -420,3 +421,164 @@ paths:
           description: 'Successful query'
         '404':
           description: 'Invalid input'
+  /gotenzymes/reactions/{reactionId}:
```

Comment:
Do we want to rename this to `keggId`? This also applies for `compound`, `organism` and `gene`.

Reply:
For clarity, that would be of help, but it would also highlight the bug for the genes where our URL expects only a subpart of the gene id.

Reply:
I really don't have a strong opinion of this, feel free to commit changes as you see fit

Reply:
I'm fine with how things are and sounds like it's not clear that it's better so I will resolve the conversation for now but feel free to un-resolve if you disagree.